framework/src/modules/chain/rounds/round.js

```diff
@@ -303,32 +303,26 @@ class Round {
 		if (exceptions.rounds[this.scope.round.toString()]) {
 			// Apply rewards factor
 			roundRewards.forEach((reward, subIndex) => {
-				roundRewards[subIndex] = new Bignum(reward.toPrecision(15))
-					.multipliedBy(
-						exceptions.rounds[this.scope.round.toString()].rewards_factor
-					)
-					.integerValue(Bignum.ROUND_FLOOR);
+				roundRewards[subIndex] = new BigNum(reward.toPrecision(15))
+					.times(exceptions.rounds[this.scope.round.toString()].rewards_factor)
+					.floor(BigNum.ROUND_FLOOR);
 			});
 
 			// Apply fees factor and bonus
-			roundFees = new Bignum(roundFees.toPrecision(15))
-				.multipliedBy(
-					exceptions.rounds[this.scope.round.toString()].fees_factor
-				)
+			roundFees = new BigNum(roundFees.toPrecision(15))
+				.times(exceptions.rounds[this.scope.round.toString()].fees_factor)
 				.plus(exceptions.rounds[this.scope.round.toString()].fees_bonus)
-				.integerValue(Bignum.ROUND_FLOOR);
+				.floor(BigNum.ROUND_FLOOR);
 		}
 
-		const fees = new Bignum(roundFees.toPrecision(15))
+		const fees = new BigNum(roundFees.toPrecision(15))
 			.dividedBy(ACTIVE_DELEGATES)
-			.integerValue(Bignum.ROUND_FLOOR);
-		const feesRemaining = new Bignum(roundFees.toPrecision(15)).minus(
-			fees.multipliedBy(ACTIVE_DELEGATES)
+			.floor(BigNum.ROUND_FLOOR);
+		const feesRemaining = new BigNum(roundFees.toPrecision(15)).minus(
+			fees.times(ACTIVE_DELEGATES)
 		);
 		const rewards =
-			new Bignum(roundRewards[index].toPrecision(15)).integerValue(
-				Bignum.ROUND_FLOOR
-			) || 0;
+			new BigNum(roundRewards[index].toPrecision(15)).floor() || 0;
```

Comment:
Don't we need `BigNum.ROUND_FLOOR` here as in the other uses of `.floor`?

```suggestion
			new BigNum(roundRewards[index].toPrecision(15)).floor(BigNum.ROUND_FLOOR) || 0;
```

Reply:
In the library floor() does not take any arguments.

Reply:
https://github.com/LiskHQ/bignum/blob/804cff6af6f8f840663a361ef36f031979003209/bignum.js#L1018

Reply:
Hmm actually 0 decimal places will check more

Reply:
Yeah it's good as before it was returning `integerValue()` that as per docs `Returns a BigNumber whose value is the value of this BigNumber rounded to an integer using rounding mode` so `floor()` is correct here